libs/dh/message-archive/feature-log-search/src/lib/dh-message-archive-log-search.component.html

```diff
@@ -16,35 +16,26 @@
 -->
 
 <ng-container *transloco="let transloco; read: 'messageArchive.search'">
-  <h1>{{ transloco('title') }}</h1>
+  <h1>{{ transloco("title") }}</h1>
   <form #searchForm name="log-search-form" (ngSubmit)="onSubmit()">
     <div class="wrapper">
       <watt-form-field size="normal">
         <watt-label class="watt-text-m">{{
-          transloco('documenttype')
+          transloco("documenttype")
         }}</watt-label>
 
-        <select
-          #searchInputDocumentType
-          wattInput
-          name="documentType"
+        <watt-dropdown
           [(ngModel)]="this.searchCriteria.rsmName"
+          [formControl]="rsmFormControl"
```

Comment:
issue: it's not recommended to use `ngModel` and `formControl` on the same field. The `watt-dropdown` component works with either, so you can remove one of them. By looking at the file as a whole, you're mostly using the template-driven approach, so you can delete the `formControl`. Both here and the `rsmFormControl` property.